docs/semgrep-supply-chain/overview.md

```diff
@@ -21,121 +21,112 @@ Object.entries(frontMatter).filter(
+introduced by open-source dependencies, using high-signal rules to determine the
+vulnerability's [reachability](/semgrep-supply-chain/glossary/#reachability).
+
+To do this, Semgrep Supply Chain parses lockfiles for a list of dependencies,
+then scans your codebase using [rules](/running-rules/) that specify the
+following information:
 
-* The range of versions that contain the dependency's vulnerability.
-* A pattern for vulnerable code, such as passing in unsanitized data.
+* The dependency versions that contain a vulnerability;
+* The pattern for the vulnerable code that Semgrep compares against your code;
 * The severity of the vulnerability.
 
-Semgrep Supply Chain generates a **finding** when it detects a match. If the dependency's version is within the range and finds the matching code within your codebase, the finding is **reachable**.
+The following diagram shows the relationship between a Semgrep Supply Chain
+rule, the codebase scanned, and its lockfile:
 
-A finding is **unreachable** if the dependency contains a known vulnerability, but the vulnerable matching code is not used in your codebase.
+![Relationship between a Semgrep Supply Chain rule, lockfile, CVE record, and codebase](/img/sc-reachability-analysis.png)
+_Figure 2_. Relationship between a Supply Chain rule, lockfile, CVE record, and codebase.
 
-A finding is **undetermined** if Semgrep detects a match for the dependency in the vulnerable version range, but does not have a reachability rule for this vulnerability. This is most common with older or lower severity vulnerabilities.
+### Types of Semgrep Supply Chain findings
 
-Undetermined findings are generated by Semgrep Supply Chain **[lockfile-only](/semgrep-supply-chain/glossary/#lockfile-only-rules)** rules, which are rules that do not perform reachability analysis. These findings are generated by checking the dependency's version in its lock or manifest file against versions with known vulnerabilities. These rules are on par with [GitHub Dependabot](https://github.com/dependabot).
+If Semgrep Supply Chain determines that the codebase leverages a package
+containing a vulnerability, it generates a **finding**. If Semgrep Supply Chain
+also determines that there's a code pattern in the codebase that matches the
+vulnerability definition, then the finding is flagged as **reachable**
+(otherwise, it is **unreachable**).
 
-In Semgrep Cloud Platform, specific findings of a dependency and code match are called **usages**. Usages are grouped by their **vulnerability**. Vulnerabilities in Semgrep Supply Chain typically have a CVE number corresponding to the record in the [CVE Program](https://www.cve.org/About/Overview).
+If Semgrep Supply Chain detects the use of a vulnerable version of a dependency,
+but it doesn't have a reachability rule, then the finding is **undetermined**
+(this is most common with older vulnerabilities or vulnerabilities with lower
```

Comment:
What's our source for this?

Reply:
This information comes from line 41 of the original version of the doc; I also found [this Slack conversation that supports this](https://semgrepinc.slack.com/archives/C032JB2HRDK/p1701365436914159)

Reply:
Our [exact date is May 2022](https://semgrep.dev/docs/supported-languages/#semgrep-supply-chain) for reachability rules - let's use something concrete?